### PR TITLE
Add a javascript file to capture UTM data from query params and save them into a cookie.

### PR DESCRIPTION
**Pull request by matthewchigira**
Closes #2216 (APM)

Related PR in APM repo:
https://github.com/scoutapp/apm/pull/2242

### CODE REVIEW

**Comment by matthewchigira** [comment]
For some reason Travis CI is set up to test on Ruby 2.2.5 (passes), 2.3.3 (fails) and 2.40 (fails) but the Ruby version is locked to 2.2.2...

File: source/javascripts/app/_utm.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+window.addEventListener("DOMContentLoaded", function(event) {
+  var searchParams = new URLSearchParams(window.location.search);
```
Comment (cschneid) [inline source/javascripts/app/_utm.js]: This is not supported at all on Internet Explorer. I don't care much about capturing UTM in that case, but we should wrap this in a try/catch to not just break the rest of the page.

Comment (matthewchigira) [reply source/javascripts/app/_utm.js]: I experimented with adding a try/catch block and providing a random string for the event name, but neither Firefox or Chrome throw the exception and the catch block never executes. The page load continues ok after this (obviously without adding the cookies). This leads me to believe that maybe adding an event listener that doesn't exist doesn't throw an exception?

Comment (matthewchigira) [reply source/javascripts/app/_utm.js]: Added a try/catch block for IE

**Pull request merged**